fem/plinearform.hpp

```diff
@@ -92,6 +92,25 @@ class ParLinearForm : public LinearForm
        @note This method does not perform assembly. */
    void Update(ParFiniteElementSpace *pf, Vector &v, int v_offset);
 
+
+   /** @brief Make the ParLinearForm reference external data on a new
+       FiniteElementSpace. */
+   /** This method changes the FiniteElementSpace associated with the ParLinearForm
+       to @a *f and sets the data of the Vector @a v (plus the @a v_offset) as external
+       data in the ParLinearForm.
+       @note This version of the method will also perform bounds checks when
+       the build option MFEM_DEBUG is enabled. */
+   virtual void MakeRef(FiniteElementSpace *f, Vector &v, int v_offset);
```

Comment:
Hey @stefanhenneking, 

what is the purpose of this method (with this particular signature)? Is it really needed ?

Reply:
@stefanhenneking , never mind...disregard the previous question. 
However, shouldn't there be a check that the dynamic cast is not null, i.e, pfes is a ParFiniteElementSpace?

Reply:
Good catch! I'll add an assert statement to check that `pfes` is not `NULL` after casting.